docs/source/analyze/certificates.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Certificates
+description: Perform analysis and checks on Certificates found in the cluster
+---
+
+The `certificates` analyzer alerts users when a specific certificate is either invalid or nearing its expiration date.
+This analyzer's outcome `when` clause compares the condition specified with the resources present on the certificates.
+
+The `when` value in an outcome of this analyzer contains the certificates that match the filters, if any filters are defined.
+If there are no defined filters, the `when` value contains all certificates in the cluster.
```

Comment:
Can we word this such that we don't suggest that the bundle contains the entire certificate?  (it doesn't, right?)

Reply:
Yes, it will not contains entire certificates
changed to 
```
If there are no defined filters, it will based on validity of the certificate. For pass outcomes, the certificate which is valid will be matched. For fail outcomes, the certificate which is invalid will be matched.
```
to make it more clear.